app/views/overview.html

```diff
@@ -12,38 +12,9 @@
 
 {% include "includes/sub-nav.html" %}
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <table>
-    <tbody>
-        <tr>
-        <th scope="row">Grade</th>
-        <td class="sln-grade numeric">{{ overviewDetails.grade }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Team</th>
-        <td class="numeric"><a href="/team/{{ overviewDetails.teamId }}">{{ overviewDetails.teamName }}</a></td>
-        </tr>
-        <tr>
-        <th scope="row">Capacity</th>
-        <td class="numeric"> {{ overviewDetails.capacity | round }}%</td>
-        </tr>
-        <tr>
-        <th scope="row">Cases</th>
-        <td class="numeric">{{ overviewDetails.cases }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Contracted hours</th>
-        <td class="numeric">{{ overviewDetails.contractedHours }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Reduction</th>
-        <td class="numeric">{{ overviewDetails.reduction }}</td>
-        </tr>
-    </tbody>
-    </table>
-  </div>
-</div>
+{% include "includes/individual-overview.html" %}
+
+{% include "includes/team-overview-table.html" %}
```

Comment:
I see the team view being extended so it can display any table which is a higher level than an individual. It might be worth having an if/case statement here so it's obvious that only one partial is actually being loaded.